### PR TITLE
EHT-534 Update bridge node version and configs

### PR DESCRIPTION
**Pull request by kowshikvajrala**

### CODE REVIEW

File: group_vars/all.yml [hunk 1]
```diff
@@ -21,6 +21,10 @@ go:
   path: "{{ tmp_dir }}/go"
   module111: "on"
 
+# zfs
+zfs_image_size: 50G # Update according to server
```
Comment (aditya-manit) [inline group_vars/all.yml:25]: Maybe we should use an external variable here and default it to 50, just like snapshot url

File: roles/bridge/tasks/main.yml [hunk 2]
```diff
@@ -31,6 +32,7 @@
     - Restart service
 
 - name: Fetch secrets from HCP Vault and save to files
+  when: fetch_validator_keys | default(false) | bool
```
Comment (aditya-manit) [inline roles/bridge/tasks/main.yml:35]: lets rename to fetch_bridge_keys or we can omit this variable, a bridge instance can run with two keys, so this variable isnt really needed

File: roles/bridge/tasks/setup_zfs.yml [hunk 3]
```diff
@@ -32,18 +32,22 @@
...
   changed_when: false
+  args:
+    executable: /bin/bash
 
 - name: Create ZFS pool with mount point
   ansible.builtin.command:
     cmd: zpool create -m {{ bridge_zfs_pool_mount_point }} {{ bridge_zfs_pool_name }} {{ loop_device.stdout }}
-    creates: "{{ bridge_zfs_pool_mount_point }}"
-  when: loop_device.stdout is defined
+  changed_when: false
```
Comment (aditya-manit) [inline roles/bridge/tasks/setup_zfs.yml:46]: I think creates is better, it sort of defines what do we expect this command to change or do

**Review by aditya-manit — CHANGES_REQUESTED** [review]
__

File: roles/bridge/tasks/setup_zfs.yml (hunk 3 above)
Comment (aditya-manit) [reply roles/bridge/tasks/setup_zfs.yml:46]: Any reason to remove it ?

Comment (kowshikvajrala) [reply roles/bridge/tasks/setup_zfs.yml:46]: It's creating the creates folder and skipping the creation of zpool

File: roles/bridge/tasks/main.yml (hunk 2 above)
Comment (kowshikvajrala) [reply roles/bridge/tasks/main.yml:35]: Ok

File: group_vars/all.yml (hunk 1 above)
Comment (kowshikvajrala) [reply group_vars/all.yml:25]: Ya mentioned it in the readme

Comment (kowshikvajrala) [reply group_vars/all.yml:25]: Pass it as extra variable

File: README.md [hunk 4]
```diff
@@ -63,23 +77,52 @@ This playbook allows customization through several variables. You can define the
...
 
-3. Then run the playbook:
+3. Configure your remote server username and private key file path in the `ansible.cfg` file. Additionally, set the SSH port for your server by adjusting the `ansible_port` variable in `group_vars/all.yml`.
 
-  ```bash
-  ansible-playbook setup_bridge.yml -l bridge.celestia.testnet.encapsulate.xyz
+4. Then run the playbook:
+
+  ```
+  ansible-playbook setup_bridge.yml -l bridge.celestia.testnet.encapsulate.xyz -e "fetch_validator_keys=true zfs_setup=true zfs_image_size=50G zfs_compression_level=zstd-3
```
Comment (aditya-manit) [inline README.md:120]: lets update the readme to and remove fetch_validator_keys=true

**Review by aditya-manit — APPROVED** [review]

**Pull request merged**